pyrk/th_component.py

```diff
@@ -93,7 +93,7 @@ def dtemp(self):
         if self.prev_t_idx == 0:
             return 0.0*units.kelvin
         else:
-            return (self.T[self.prev_t_idx] - self.T[self.prev_t_idx-1])
+            return (self.T[self.prev_t_idx] - self.T0)
```

Comment:
This is good. But it is going to be changed to some steady state temperature when we merge the branch with_units in, which is even better for computation stability.